--- jdk8-openjdk-src-8u-202/hotspot/src/os/linux/vm/os_linux.cpp~	2019-02-01 21:52:41.000000000 +0000
+++ jdk8-openjdk-src-8u-202/hotspot/src/os/linux/vm/os_linux.cpp	2019-05-18 01:27:33.360945174 +0000
@@ -3532,7 +3532,7 @@
 
     // Don't try UseTransparentHugePages since there are known
     // performance issues with it turned on. This might change in the future.
-    UseTransparentHugePages = false;
+    UseTransparentHugePages = true;
   }
 
   if (UseTransparentHugePages) {
--- jdk8-openjdk-src-8u-202/hotspot/make/linux/Makefile~	2019-02-01 21:52:41.000000000 +0000
+++ jdk8-openjdk-src-8u-202/hotspot/make/linux/Makefile	2019-05-18 01:32:54.131940708 +0000
@@ -233,7 +233,7 @@
 # Solaris 2.5.1, 2.6).
 # Disable this check by setting DISABLE_HOTSPOT_OS_VERSION_CHECK=ok.
 
-SUPPORTED_OS_VERSION = 2.4% 2.5% 2.6% 3% 4%
+SUPPORTED_OS_VERSION = 2.4% 2.5% 2.6% 3% 4% 5%
 OS_VERSION := $(shell uname -r)
 EMPTY_IF_NOT_SUPPORTED = $(filter $(SUPPORTED_OS_VERSION),$(OS_VERSION))
 
